.eslintrc.js

```diff
@@ -1,25 +1,32 @@
 module.exports = {
-  'extends': 'airbnb',
-  'env': {
-    'mocha': true,
-    'browser': true,
-    'worker': true,
-    'node': true
+  extends: 'airbnb-base',
```

Comment:
The [`airbnb-base` config](https://www.npmjs.com/package/eslint-config-airbnb-base) doesn't require the React and JSX plugins.  Since those were unused in this project, they are removed from the dependencies below.